default/scripting/policies/EXOBOT_PRODUCTIVITY.focs.txt

```diff
@@ -19,9 +19,10 @@ Policy
                 TargetIndustry low = 0
                 Focus type = "FOCUS_INDUSTRY"
             ]
+            // Very bad multiplier is applied before, so it must be reverted before applying bad multiplier.
             effects = SetTargetIndustry value = Value
                         * (NamedReal name = "PLC_EXOBOT_PRODUCTIVITY_TARGET_INDUSTRY_SCALING"
-                                     value = [[BAD_MULTIPLIER]] - [[VERY_BAD_MULTIPLIER]])
+                                     value = ([[BAD_MULTIPLIER]] / [[VERY_BAD_MULTIPLIER]])
```

Comment:
needs another `)` at the end, I think...

Reply:
Of course you are right. That's the problem with proposing single line changes: it's hard to read it in context
```suggestion
                                     value = ([[BAD_MULTIPLIER]] / [[VERY_BAD_MULTIPLIER]]))
```